processor/intervalprocessor/processor.go

```diff
@@ -109,16 +122,16 @@ func (p *Processor) ConsumeMetrics(ctx context.Context, md pmetric.Metrics) erro
 						return false
 					}
 
-					mClone, metricID := p.getOrCloneMetric(rm, sm, m)
-					aggregateDataPoints(m.Summary().DataPoints(), mClone.Summary().DataPoints(), metricID, p.summaryLookup)
+					mClone, metricID, partition := p.getOrCloneMetric(rm, sm, m)
```

Comment:
Do we want to partition on metricID or streamID? (cc @sh0rez)

The current code partitions on metricID. Which is much simpler code-wise. But I do worry that it won't be enough. IE, if all the data are the same metrics, but only differentiated by datapoint labels. They'll all be partitioned into the same bucket, and thus defeat the purpose.

Reply:
friendly ping @sh0rez :)